roles/openstack_create_server/vars/main.yml

```diff
@@ -6,30 +6,21 @@ ec2_flavors:
   - { name: m1.medium, vcpu: 1, memory: 3840, disk: 96 }
   - { name: m1.large, vcpu: 2, memory: 7680, disk: 128 }
   # https://aws.amazon.com/ec2/instance-types/#m4
-  - { name: m4.large, vcpu: 2, memory: 8192, disk: 100 }
-  - { name: m4.xlarge, vcpu: 4, memory: 16384, disk: 200 }
-  - { name: m4.2xlarge, vcpu: 8, memory: 32768, disk: 300 }
-  - { name: m4.4xlarge, vcpu: 16, memory: 65536, disk: 400 }
-  - { name: m4.10xlarge, vcpu: 40, memory: 163840, disk: 500 }
+  - { name: m4.large, vcpu: 2, memory: 8192, disk: 96 }
```

Comment:
These may have to be lowered just a little bit.  i.e. reduce each size by 256MB?